.azure/bicep/config/web-app-settings.bicep

```diff
@@ -0,0 +1,45 @@
+param keyVaultName string
+param appInsightsName string
+param appName string
+param appSettings object = {}
+
+var secretRef = '@Microsoft.KeyVault(VaultName=${keyVaultName};SecretName='
+
+var sharedSettings = {
+  OutboundEmail__FromName: 'hutch'
+  OutboundEmail__Provider: 'sendgrid'
+  OutboundEmail__SendGridApiKey: '${secretRef}sendgrid-api-key)'
+  OutboundEmail__FromAddress: 'noreply@hutch.com'
+  // Environment for deployment (Development or Production)
+  ASPNETCORE_ENVIRONMENT: '${secretRef}ASPNETCORE_ENVIRONMENT)'
+  // Paths to SSL certificates
+  ASPNETCORE_Kestrel__Certificates__Default__Path: '${secretRef}ASPNETCORE_Kestrel__Certificates__Default__Path)'
+  ASPNETCORE_Kestrel__Certificates__Default__KeyPath: '${secretRef}ASPNETCORE_Kestrel__Certificates__Default__KeyPath)'
+  // Connection string to database
+  ASPNETCORE_ConnectionStrings__Default: '${secretRef}ASPNETCORE_ConnectionStrings__Default)'
+  // Credentials for RabbitMQ queue
+  ASPNETCORE_JobQueue__HostName: '${secretRef}ASPNETCORE_JobQueue__HostName)'
```

Comment:
I don't think this is a secret.

Also, are we using RabbitMQ in Azure? Or Azure Queue Storage